collection_json.py

```diff
@@ -140,6 +140,29 @@ class Template(ComparableObject):
 
     """Object representing a Collection+JSON template object."""
 
+    @staticmethod
+    def from_json(data):
+        """Return a template instance.
+
+        Convenience method for parsing 'write' responses which should only contain the template,
+        not in a collection object.
```

Comment:
Wording is a little confusing, did some re-wording to simply say it should only contain a template object.